create_infotexts.py

```diff
@@ -8,88 +8,89 @@
 Outputs <Fotonummer>.info files in a subdirectory ./infofiles/
 """
 
-import os
 import json
 import re
 import pandas as pd
 import batchupload.helpers as helpers
 
 people_mapping_string = """{
```

Comment:
I would say that that bit should probably be broken out anyway since it is not related to the same things as the rest of this file. But that can be discussed in that task/commit instead =)